web/backend/src/Server.ts

```diff
@@ -175,78 +176,98 @@ app.get('/api/personal_info', (req, res) => {
   }
 });
 
-function isAuthorized(roles: string[], req: express.Request): boolean {
-  if (!req.session || !req.session.userid) {
-    return false;
-  }
+// function isAuthorized(roles: string[], req: express.Request): boolean {
+// return true;
+// if (!req.session || !req.session.userid) {
+// return false;
+// }
 
-  const { role } = req.session;
+// const { role } = req.session;
 
-  return roles.includes(role as string);
-}
+// return roles.includes(role as string);
+// }
+
+const e = newEnforcer('model.conf', 'policy.csv');
 
 // ---
 // Users role
 // ---
-
 // This call allow a user that is admin to get the list of the people that have
 // a special role (not a voter).
-app.get('/api/user_rights', (req, res) => {
-  if (!isAuthorized(['admin'], req)) {
-    res.status(400).send('Unauthorized - only admins allowed');
-    return;
-  }
-
-  const opts: RangeOptions = {};
-  const users = Array.from(
-    usersDB.getRange(opts).map(({ key, value }) => ({ id: '0', sciper: key, role: value }))
-  );
-  res.json(users);
+app.get('/api/user_rights', async (req, res) => {
+  e.then((enforcer) =>
```

Comment:
if you have an async function, you don't need to do `.then` and `.catch`, you can just use `await` where you would use a then, and `try/catch` instead of `.catch`.

see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await